jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/ExtendedPagedTable.java

```diff
@@ -180,10 +195,12 @@ public boolean clearCurrentSelection(CellPreviewEvent<T> event) {
                             final T domain = event.getValue();
                             if (domain.isSelected()) {
                                 input.setChecked(false);
-                                domain.setSelected(false);
+                                setItemSelection(domain,
+                                                 false);
```

Comment:
I'm not completely sure if ```domain.isSelected()``` and ```isItemSelected(item)``` are interchangeable here.
If they are, then instead of ```setItemSelection(domain, false)``` a direct call to ```selectedItems.remove(item)``` can be made here.

The same comment applies to ```else``` branch in this case.

Reply:
@bsremac domain.isSelected is the new value that comes in the click on checkbox event. isItemSelected specifies if the item has been previously selected. SetItemSelection tries to encapsulate the interaction with that saved selections.